driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/MapReducePublisherImpl.java

```diff
@@ -104,12 +104,14 @@
     }
 
     @Override
+    @SuppressWarnings("deprecation")
```

Comment:
See above.  Shouldn't be necessary

Reply:
The warnings still need to be suppressed - but have added it to the class which follows existing convention.